### PR TITLE
Flux Teams package

### LINKED ISSUE #1064: Document Flux 'teams' package
PR: https://github.com/influxdata/flux/pull/2866

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1064 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@sranka, would love your review on this PR as well.

File: content/v2.0/reference/flux/stdlib/contrib/teams/message.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: teams.message() function
+description: >
+  The `teams.message()` function sends a single message to a Microsoft Teams channel using
+  a [incoming webhook](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks&amp?page=3).
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/teams/message.md:5]: should this be URL on line 5?:
https://docs.microsoft.com/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

**Comment by sranka** [comment]
@sanderson I don't have permission to formally approve this, but it looks good to me

File: content/v2.0/reference/flux/stdlib/contrib/teams/message.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    |> range(start: -1m)
+    |> filter(fn: (r) => r._measurement == "statuses")
+    |> last()
+    |> findRecord(fn: (key) => true, idx: 0)
+
+teams.message(
+  url: "https://outlook.office.com/webhook/example-webhook",
+  title: "Disk Usage"
+  text: "Disk usage is: *${lastReported.status}*.",
+  summary: "Disk Usage is ${lastReported.status}"
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/contrib/teams/message.md:70]: ditto

File: content/v2.0/reference/flux/stdlib/contrib/teams/endpoint.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+crit_statuses = from(bucket: "example-bucket")
+  |> range(start: -1m)
+  |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
+
+crit_statuses
+  |> endpoint(mapFn: (r) => ({
+      title: "Disk Usage"
+      text: "Disk usage is: **${r.status}**.",
+      summary: "Disk Usage is ${r.status}"
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/contrib/teams/endpoint.md]: These are literal strings that get sent to a teams channel, so the casing is likely what users will use. I did change "Usage" to lowercase in the summary though.

File: content/v2.0/reference/flux/stdlib/contrib/teams/message.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/contrib/teams/message.md]: Yes, that way it shows up with this doc appears in the a list generated by the children shortcode. The link html tag gets stripped out when it builds the page meta description.

**Pull request merged**